client/locations-list-map.js

```diff
@@ -672,14 +672,20 @@ function getMarkers(data, appliedFilters, bounds, markerOptions) {
         if (hasEntryFilter) {
           if (!Object.keys(entryAge).some((entryFilter) => {
             const rangeArray = entryFilter.split('-');
-            const min = parseInt(rangeArray[0], 10);
-            if (rangeArray.length === 2 && entry.entry_age >= min && entry.entry_age <= parseInt(rangeArray[1], 10)) {
-              return true;
+            const entryDate = new Date(entry.timestamp);
+            const minDate = new Date();
+            minDate.setDate(minDate.getDate() - parseInt(rangeArray[0], 10));
```

Comment:
This is okay for the current codebase, but we're getting a little close to using a human-readable string as a computer-parseable value. That tends to be more fragile than, say, having a map of the exact value to a number because it locks the string format into a structure in a way that has no statically verifable or searchable indication.